packages/flutter/lib/src/gestures/velocity_tracker.dart

```diff
@@ -65,8 +65,8 @@ class Velocity {
   bool operator ==(dynamic other) {
     if (other is! Velocity)
       return false;
-    final Velocity typedOther = other;
-    return pixelsPerSecond == typedOther.pixelsPerSecond;
+    return other is Velocity
```

Comment:
Looks like the if above could be removed now?

Reply:
good catch. Done